R/kelp.R

```diff
@@ -10,13 +10,17 @@ kelp <- R6::R6Class(
   cloneable = FALSE,
 
   public = list(
+    #' @field seaweed_url Base URL for seaweed master.
+    seaweed_url = NULL,
+
     #' @description
     #' Create client object for sending http requests to seaweed
     #'
     #' @param seaweed_url Root URL of Seaweed
     #'
     #' @return A new `kelp` object
     initialize = function(seaweed_url) {
+      self$seaweed_url <- seaweed_url
```

Comment:
Sometimes I follow this up with `lockBinding("seaweed_url", self)` which gives a read-only field